roles/tofu/tasks/plan.yml

```diff
@@ -14,13 +14,13 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: "Tofu Action: '{{ cifmw_tofu_plan_state }}'"
+- name: "Tofu Action"
   environment:
     TF_CLI_ARGS: '-no-color'
     TF_IN_AUTOMATION: 'true'
   vars:
     tf_project_path: "{{ cifmw_tofu_remote_execution | ternary(cifmw_tofu_project_temp_dir['path'], cifmw_tofu_project_path) }}"
-  community.general.cloud.misc.terraform:
+  terraform:
```

Comment:
Why is this not working? The Ansible recommendation is to use the FQN of each/role module.

Reply:
@eshulman2 hello, care to check that comment out? FQCN are mandatory in ansible, especially for non-builtin modules.

Reply:
changed